src/app/sitting-records/manage-sitting-records/manage-sitting-records.component.html

```diff
@@ -0,0 +1,141 @@
+<main class="govuk-main-wrapper " id="main-content" role="main">
+    
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-3">Manage Judicial Sitting Records</h1>
+    <h1 class="govuk-body-l govuk-!-margin-bottom-5">Find, add or delete judicial sitting records</h1>
+
+    <form 
+        [formGroup]="manageRecords"
+        (ngSubmit)="submitForm()"
+    >
+
+        <div class="govuk-form-group govuk-!-margin-bottom-3">
+            <label class="govuk-label" for="sort">
+              Select a tribunal service 
+            </label>
+            <select class="govuk-select govuk-!-width-two-thirds" 
+                formControlName="tribunalService"
+                id="sort" name="sort"
+            >
+              <option [ngValue]="''" disabled selected>--Select a value--</option>
+              <option [ngValue]="'1'">1</option>
+              <option [ngValue]="'2'">2</option>
+
+            </select>
+          </div>
+
+          <div class="govuk-form-group govuk-!-margin-bottom-3"                 
+          [class.govuk-form-group--error] = "f['venue'].errors !== null && f['venue'].dirty && f['venue'].touched">
+            <label class="govuk-label" for="venue-select">
+              Select a venue
+            </label>
+            <p id="msr-require-match" class="govuk-error-message" *ngIf="f['venue'].errors && f['venue'].dirty && f['venue'].touched">
+              <span class="govuk-visually-hidden">Error:</span> Select valid Venue 
+            </p>
+            <div class="auto-complete-container">
+              <input class="govuk-input govuk-!-width-two-thirds"
+                formControlName="venue" id="venue-select" 
+                name="venue-select" type="text" 
+                placeholder="Start typing to search for a venue"
+                [matAutocomplete]="autoSearchVenue"
+                required>
+              <mat-autocomplete class="mat-autocomplete-panel-extend" autoActiveFirstOption #autoSearchVenue="matAutocomplete" [displayWith]="showVenue">
```

Comment:
We should change the autocomplete to use (optionSelected) instead of (onSelectionChange), this way we can prevent the error popping up when user selects an option

Reply:
Done